src/test/java/edu/ucsb/cs156/gauchoride/controllers/RideControllerTests.java

```diff
@@ -276,15 +282,17 @@ public void test_that_logged_in_driver_can_get_by_id_when_the_id_exists() throws
                 long otherUserId = userId + 1;
 
                 Ride ride = Ride.builder()
-                                .riderId(otherUserId)
-                                .student("DGaucho")
+                                .riderId(userId)
+                                .student("CGaucho")
                                 .day("Monday")
                                 .course("CMPSC 156")
                                 .startTime("2:00PM")
                                 .endTime("3:15PM")
                                 .dropoffLocation("South Hall")
+                                .dropoffRoom("1431")                    //MODIFIED
```

Comment:
please proceed as above

Reply:
Thanks for pointing that out. I've deleted them in the latest push. Please give it a second look.